Library/Formula/mongrel2.rb

```diff
@@ -13,6 +13,8 @@ class Mongrel2 < Formula
     sha256 "7cdef65628b457f03f3eb76fcf74276ffbc4b7d9fa0ff6d017fa43a8db98eeca" => :mavericks
   end
 
+  patch :DATA
```

Comment:
Presumably don't want to apply this patch to `head` then? Can you also add a small comment above the patch as to what it's doing and a link to the upstream commit (Or even use the upstream commit as an external patch).